build_tools/cmake/CMakeLists.txt

```diff
@@ -42,3 +46,17 @@ install(FILES
   DESTINATION ${_cmake_installdir}
   COMPONENT IREECMakeExports
 )
+
+file(GLOB _iree_cmake_helpers "${CMAKE_CURRENT_SOURCE_DIR}/iree_*.cmake")
+list(APPEND
+  _iree_cmake_helpers
+  ${CMAKE_CURRENT_SOURCE_DIR}/external_cc_library.cmake
+  ${CMAKE_CURRENT_SOURCE_DIR}/flatbuffer_c_library.cmake
+)
+list(REMOVE_ITEM _iree_cmake_helpers iree_llvm.cmake)
+
+install(FILES
+   ${_iree_cmake_helpers}
+   DESTINATION ${_cmake_installdir}
+   COMPONENT IREECMakeExports
+)
```

Comment:
These CMake files are also private: https://github.com/iree-org/iree/blob/main/build_tools/cmake/README.md

If the project is complex enough that out of tree usage feels the need to use these CMake helpers, _that's_ what needs to be fixed, _not_ making the helper functions public. Downstream users of the C/C++ code should be able to use any build system they want, not just CMake.

Reply:
The goal here was to be able to do `iree-install-dist` and have enough to build tools (such as `iree-opt`): https://github.com/makslevental/iree-standalone